Automattic/woocommerce-payments
# Re-apply update customer optimization

Fixes #5417 

## Changes proposed in this Pull Request









- Make the request to update customer info asynchronous.

## Testing instructions






1. Set a breakpoint [here](https://github.com/Automattic/woocommerce-payments/blob/f2b19a3b06612b7225c3e6ad460f27d8fe205696/includes/class-wc-payment-gateway-wcpay.php#L3318).
2. Add a product to your cart, complete a payment.
3. Make sure the customer is redirected to the **Order Received** page after a successful payment even if you're still on the breakpoint.


-------------------


**Post merge**
